app/controllers/search_controller.rb

```diff
@@ -2,11 +2,30 @@ class SearchController < ApplicationController
   def index
     return if params[:query].nil?
 
-    @results = Person.search(params[:query])
-    @results += Room.search(params[:query])
-    @results += Chair.search(params[:query])
+    @exact_results = add_results_for(params[:query])
+
+    words_in_query = params[:query].scan(/[A-Za-z0-9]+/)
```

Comment:
so you don't parse points for example? 

In addition [A-Za-z0-9] is the same as \w

Reply:
\w includes underscore, I would leave it as it is.

I think, we shouldn't parse Points. That way we still get results if points are not set correctly e.g `Dr.Perscheid` instead of `Dr. Perscheid`